myproject/supermarket_manager/views.py

```diff
@@ -494,6 +496,8 @@ def update_account(request, account_id):
             account.status = data.get("status", account.status)
 
         elif request.method == "PATCH":
+            # Change password
```

Comment:
Should comment above the `elif` statement.

Reply:
Done